From 80ecf7edc4659c580fcd9d752b57bf23ef7399c0 Mon Sep 17 00:00:00 2001
From: Kyle Brenneman <kbrenneman@nvidia.com>
Date: Mon, 26 Aug 2024 09:19:15 -0600
Subject: [PATCH 3/9] Fix a declaration after code.

In wayland-egldisplay.c, move a declaration to the top of the function.
Older versions of C don't allow mixing declarations and code.
---
 src/wayland-egldisplay.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/wayland-egldisplay.c b/src/wayland-egldisplay.c
index 4d25cd7..468b32d 100644
--- a/src/wayland-egldisplay.c
+++ b/src/wayland-egldisplay.c
@@ -568,9 +568,10 @@ static void
 dmabuf_feedback_check_done(void *data, struct zwp_linux_dmabuf_feedback_v1 *dmabuf_feedback)
 {
     WlServerProtocols *protocols = (WlServerProtocols *)data;
+    drmDevice *drm_device;
+
     (void) dmabuf_feedback;
 
-    drmDevice *drm_device;
     assert(getDeviceFromDevId);
     if (getDeviceFromDevId(protocols->devId, 0, &drm_device) == 0) {
         if (drm_device->available_nodes & (1 << DRM_NODE_RENDER)) {
-- 
2.43.0

